serverless/src/forwarder.ts

```diff
@@ -206,9 +210,30 @@ export async function getExistingLambdaLogGroupsOnStack(cloudWatchLogs: CloudWat
   return logGroups ?? [];
 }
 
+/*
+Grab all the subscription filters from the log group
+Loop through the subscription filters, checking if any of the filters is a existing datadog-cloudformation-macro filter
+If we did not find a datadog-cloudformation-macro filter and the number of active subscription filters on the log group 
+is less than the maxAllowableLogGroupSubscriptions from AWS cloudwatch log groups then we subscribe.
+Otherwise we do not subscribe.
+*/
```

Comment:
I would try to remove this comment -- generally I would only put a comment in code if there is something very non-obvious that the code itself doesn't explain. And even then, a brief 1-liner comment is usually better than a longer one. See if you can make your code so easy to read that no comment is needed!

Reply:
Your tests will also help document the behavior of the function. So if someone doesn't understand your code, they can look at the unit tests and see the different cases.